apps/sms/index.html

```diff
@@ -79,9 +79,9 @@
     <script defer src="views/shared/js/inter_instance_event_dispatcher.js"></script>
     <script defer src="views/shared/js/time_headers.js"></script>
     <script defer src="views/shared/js/contacts.js"></script>
+    <script defer src="views/shared/js/utils.js"></script>
```

Comment:
Utils is required by Drafts, and import-scripts trick will try to call ```importScripts`` for the Utils since it's not loaded yet - and it will fail in window context :)